src/v2/Apps/Order/Components/ShippingAddress.tsx

```diff
@@ -19,19 +20,26 @@ export const ShippingAddress = ({
     country,
     phoneNumber,
   },
+  textColor,
```

Comment:
nit you can define the default here saving the ternary operators down below, improves readability